Readme.md

```diff
@@ -41,7 +41,7 @@ If you're using GPGMail, follow these steps to install the xpc service helper.
 cp ./build/org.gpgtools.Libmacgpg.xpc.plist ~/Library/LaunchAgents/
 
 mkdir -p ~/Library/Application\ Support/GPGTools
-cp ~/build/Release/org.gpgtools.Libmacgpg.xpc ~/Library/Application\ Support/GPGTools
+cp ./build/Release/org.gpgtools.Libmacgpg.xpc ~/Library/Application\ Support/GPGTools
```

Comment:
this should also be `cp -r`, as it's a folder

Reply:
This appears to be fixed in dev.